package.json

```diff
@@ -114,7 +114,7 @@
     "@types/hammerjs": "^2.0.34",
     "@types/jasmine": "^2.5.53",
     "@types/jasminewd2": "~2.0.3",
-    "@types/jquery": "^3.3.4",
+    "@types/jquery": "^3.3.30",
```

Comment:
@joeldenning why the downgrade from 3.3.4 to 3.3.30?

Reply:
`3.3.30` is an upgrade, not an upgrade. Semantic versioning does not exactly follow mathematical `>`. Read more at https://semver.org/#semantic-versioning-specification-semver

Reply:
oh haha! well everything lgtm.